src/FilterStore.ts

```diff
@@ -0,0 +1,4 @@
+import { createSignal } from 'solid-js'
+import type { comboboxItem } from './components/Combobox'
+
+export const [selectedComboboxItems, setSelectedComboboxItems] = createSignal<comboboxItem[]>([])
```

Comment:
Can we move this to be within `Combobox.tsx`? It's typically not worth making a new file for only 4 lines of code.